cgstats.js

```diff
@@ -141,10 +141,7 @@ app.get('/search*', function(req, res) {
           .then(function(responses) {
             var users = responses.map(function(response) {
               return response.users;
-            })
-            .reduce(function(a, b) {
-              return a.concat(b);
```

Comment:
no need to concat language stats into single array. Having the stats as array of array allows to use findIndex instead of foreach.